### ISSUE TITLE
Port STK opcodes to Android [sf featuresplatforms #8]

### ISSUE THREAD

**Issue by kunstmusik** [csound/csound#208]
**Originally reported by Sourceforge User: gogins**

Port the STK plugin opcodes to the Csound 6 app for Android.

**Comment by kunstmusik** [comment]
@gogins Is this one that you intend to work on, or can we close this?

**Comment by gogins** [comment]
I am indeed going to do this.

**Comment by kunstmusik** [comment]
Okay, should we schedule for 6.08 or move to the 6.x milestone (for tickets that should be in 6.x, but not necessarily scheduled)

**Comment by gogins** [comment]
I may integrate the NDK build in Android Studio.

See: http://tools.android.com/tech-docs/new-build-system/gradle-experimental/migrate-to-stable.

**Comment by gogins** [comment]
I have added the STK opcodes to the Csound for Android app.
1. Added the STK repository to downloadDependencies.sh; clones to stk dir.
2. Created new pluginlibs/stk-csound dir with build files; the plugin is built in stk-csound/libs using sources from stk/src.
3. Modified the build-all-mkg.sh script to copy rawwaves to the app assets dir.
4. Modified the CsoundAppActivity class to, on startup:
 a. Copy all rawwaves files from the assets/rawwaves dir to the user's device Environment.DIRECTORY_MUSIC/rawwaves dir.
 b. Added CsoundAppActivity.setEnv to introspect hidden libcore.io.Libcore.setEnv function and call it to set the RAWWAVE_PATH.
5. Tested on my tablet with STKBlowBtl.csd -- works!
